addons/l10n_ar/views/account_fiscal_position_view.xml

```diff
@@ -7,7 +7,7 @@
         <field name="inherit_id" ref="account.view_account_position_form"/>
         <field name="arch" type="xml">
             <field name="auto_apply" position="after">
-                <field name="l10n_ar_afip_responsibility_type_ids" options="{'no_open': True, 'no_create': True}" attrs="{'invisible': [('auto_apply', '!=', True)]}" groups="base.group_no_one" widget="many2many_tags"/>
+                <field name="l10n_ar_afip_responsibility_type_ids" options="{'no_open': True, 'no_create': True}" attrs="{'invisible': ['|', '!', ('fiscal_country_codes', 'like', 'AR'), ('auto_apply', '!=', True)]}" groups="base.group_no_one" widget="many2many_tags"/>
```

Comment:
```suggestion
                <field name="l10n_ar_afip_responsibility_type_ids" options="{'no_open': True, 'no_create': True}" attrs="{'invisible': ['|', ('fiscal_country_codes', 'not like', 'AR'), ('auto_apply', '!=', True)]}" groups="base.group_no_one" widget="many2many_tags"/>
```
I'm really not fan of this syntax, it's not very readable/intuitive... :sweat_smile: 
You could just add the missing operators:
```diff
diff --git a/addons/web/static/src/core/domain.js b/addons/web/static/src/core/domain.js
index d2252ed2bb23..2dabe7bf4846 100644
--- a/addons/web/static/src/core/domain.js
+++ b/addons/web/static/src/core/domain.js
@@ -243,6 +243,11 @@ function matchCondition(record, condition) {
                 return false;
             }
             return fieldValue.indexOf(value) >= 0;
+        case "not like":
+            if (fieldValue === false) {
+                return false;
+            }
+            return fieldValue.indexOf(value) === -1;
         case "=like":
             if (fieldValue === false) {
                 return false;
@@ -253,6 +258,11 @@ function matchCondition(record, condition) {
                 return false;
             }
             return fieldValue.toLowerCase().indexOf(value.toLowerCase()) >= 0;
+        case "not ilike":
+            if (fieldValue === false) {
+                return false;
+            }
+            return fieldValue.toLowerCase().indexOf(value.toLowerCase()) === -1;
         case "=ilike":
             if (fieldValue === false) {
                 return false;
```
I think it's worth it and might be useful in the future :slightly_smiling_face: 
Also, using `ilike` might be a bit safer (if you agree) ?